corehq/apps/hqmedia/templates/hqmedia/partials/multimedia_js.html

```diff
@@ -1,13 +1,7 @@
 {% load hq_shared_tags %}
-{# This is fine as an inline script; multimedia needs to be reworked to work with RequireJS so we can deal with this config at that time #}
-<script>
-    YUI_config = {
-        combine: false,
-        fetchCSS: false,
-        base: "//{{ request.get_host }}/",
-        loaderPath: "static/hqmedia/MediaUploader/yui-loader.js"
-    }
-</script>
+
 <script src="{% static 'hqmedia/MediaUploader/yui-base.js' %}"></script>
+{# Once all multimedia pages are migrated this entire file can be deleted along with   YUI configuration script #}
```

Comment:
I don't understand this comment. It makes sense that this file, multimedia_js.html, can be deleted once the migration is done, but yui_config.js looks like it's still necessary for requirejs files (given that you added it to the requirejs config).